config/default.yaml

```diff
@@ -143,6 +143,12 @@ bookends:
     teardown:
         - screwdriver-artifact-bookend
 
+notification:
```

Comment:
@FenrirUnbound  It is set as `{}` in registerPlugins.js. Is that what you meant?